crates/l2/sequencer/mod.rs

```diff
@@ -37,8 +39,9 @@ pub async fn start_l2(
     rollup_store: StoreRollup,
     blockchain: Arc<Blockchain>,
     cfg: SequencerConfig,
+    cancellation_token: CancellationToken,
```

Comment:
I forgot we are reexporting CancellationToken on spawned. Let's use that reexport, instead.